webpack/assets/javascripts/react_app/components/notifications/notifications.test.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import Notifications from './';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import Store from '../../redux';
+import {
+  emptyState,
+  emptyHtml,
+  stateWithoutNotifications,
+  stateWithNotifications,
+  fullHtml,
+  stateWithUnreadNotifications,
+  serverResponse
+} from './notifications.fixtures';
+jest.unmock('jquery');
+
+const mockStore = configureMockStore([thunk]);
+
+describe('notifications', () => {
+  const $ = require('jquery');
+
+  beforeEach(() => {
+    global.__ = str => str;
+    global.tfm = {
+      tools: {
+        activateTooltips: () => {}
+      }
+    };
+
+    $.getJSON = jest.genMockFunction().mockImplementation(url => {
+      return new Promise(resolve => resolve(JSON.parse(serverResponse)));
+    });
+  });
+
+  it('empty state', () => {
+    const store = mockStore(emptyState);
+
+    const box = shallow(<Notifications store={store} />);
+
+    expect(box.render().html()).toEqual(emptyHtml);
```

Comment:
However it does catch small bugs that otherwise go un-noticed, like the time created being different on the local machine then the build machine...